packages/frontend/src/contexts/attester.js

```diff
@@ -2,25 +2,23 @@ import { makeAutoObservable } from 'mobx'
 import { SERVER } from '../config'
 
 export default class Attester {
-  epochsByAttester = []
-  epochIds = []
-  epochsByNumber = new Map()
+  epochsByAttesterId = new Map()
+  epochsById = new Map()
 
+  signUpsByAttesterId = new Map()
   signUpIds = []
   signUpsById = new Map()
   signUpsByEpoch = new Map()
 
-  attestationsIds = []
+  attestationsByAttesterId = new Map()
   attestationsById = new Map()
-  attestationsByEpoch = new Map()
-  totalPosRep = 0
-  totalNegRep = 0
-  attestationCount = 0
 
   ustIds = []
   ustById = new Map()
   ustByEpoch = new Map()
```

Comment:
I previously left these (also `loadUSTByAttester(attesterId)` and `ingestUST(_USTs)` to get data for the TODO: graph element on `AttesterPage` - which you have removed, so probably these should go too?

Reply:
Yep, agreed 356a608